server/service/src/sync/test/test_data/goods_received_line.rs

```diff
@@ -0,0 +1,118 @@
+// use crate::sync::{
+//     test::TestSyncIncomingRecord, translations::purchase_order_line::LegacyPurchaseOrderLineRow,
+// };
+// use chrono::NaiveDate;
+// use repository::PurchaseOrderLineRow;
+// use serde_json::json;
+
+// use super::TestSyncOutgoingRecord;
+
+use chrono::NaiveDate;
+use repository::{GoodsReceivedLineRow, GoodsReceivedLineRowDelete};
+use serde_json::json;
+
+use crate::sync::{
+    test::{TestSyncIncomingRecord, TestSyncOutgoingRecord},
+    translations::goods_received_line::LegacyGoodsReceivedLineRow,
+};
+
+const TABLE_NAME: &str = "goods_received_line";
```

Comment:
```suggestion
const TABLE_NAME: &str = "Goods_received_line";
```

Does this matter? I'm getting `Goods_received_line` in my mSupply sync buffer?